lib/web_ui/lib/src/engine/semantics/text_field.dart

```diff
@@ -60,6 +60,7 @@ class TextField extends RoleManager {
 
     switch (browserEngine) {
       case BrowserEngine.blink:
+      case BrowserEngine.edge:
```

Comment:
Is this always true?

Reply:
I checked it. 
(1) if the Browser is Edge and OS is IOS then the engine is Webkit. (Otherwise always blink)
(2) If browser is older than earlier 2019 it is not blink.